dev/CommandBarFlyout/CommandBarFlyoutCommandBar.h

```diff
@@ -86,4 +89,17 @@ class CommandBarFlyoutCommandBar :
     winrt::Storyboard::Completed_revoker m_closingStoryboardCompletedCallbackRevoker{};
 
     bool m_secondaryItemsRootSized{ false };
+
+    bool m_isExpandedUp{ false };
+    void AttachEventsToSecondaryStoryboards();
+
+    tracker_ref<winrt::Storyboard> m_expandedUpToCollapsedStoryboard{ this };
```

Comment:
Are these tracker refs actually needed? can we use the presense of a revoker value for a proxy for if the storyboard exists?

Reply:
Good call, i'll just stick with the revokers.